packages/forms/resources/js/components/rich-editor/extensions.js

```diff
@@ -55,98 +55,121 @@ export default async ({
     textColors,
     uploadingFileMessage,
     $wire,
-}) => [
-    Blockquote,
-    Bold,
-    BulletList,
-    Code,
-    CodeBlock,
-    CustomBlock.configure({
-        deleteCustomBlockButtonIconHtml,
-        editCustomBlockButtonIconHtml,
-        editCustomBlockUsing,
-        insertCustomBlockUsing,
-    }),
-    Details,
-    DetailsSummary,
-    DetailsContent,
-    Document,
-    Dropcursor,
-    Gapcursor,
-    Grid,
-    GridColumn,
-    HardBreak,
-    Heading,
-    Highlight,
-    HorizontalRule,
-    Italic,
-    Image.configure({
-        inline: true,
-    }),
-    Lead,
-    Link.configure({
-        autolink: true,
-        openOnClick: false,
-    }),
-    ListItem,
-    LocalFiles.configure({
-        acceptedTypes: acceptedFileTypes,
-        acceptedTypesValidationMessage: acceptedFileTypesValidationMessage,
-        get$WireUsing: () => $wire,
-        key,
-        maxSize: maxFileSize,
-        maxSizeValidationMessage: maxFileSizeValidationMessage,
-        statePath,
-        uploadingMessage: uploadingFileMessage,
-    }),
-    ...(Object.keys(mergeTags).length
```

Comment:
Some extensions have been lost from this list (grid, text color) and others now have different config like this